### PR TITLE
minor copyedits for readme

### PR DESCRIPTION
**Pull request by mmrj**
Minor copyedits while working on [SC-164866](https://app.shortcut.com/launchdarkly/story/164866/update-account-merging-guide-with-new-script).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
 ### Requirements
 
-- You must have Deno installeds
+- You must have [Deno](https://deno.land/) installed. If you use Homebrew, run `brew install deno`.
 
 
 ### Considerations
 
-- These scripts (migrate and source) are supported strictly as is and LD support cannot be expected for help running this
+- These scripts, `migrate.ts` and `source.ts`, are provided strictly as-is. LaunchDarkly Support cannot be expected for help running this.
```
Comment (ember-stevens) [inline README.md]:
```suggestion
- These scripts, `migrate.ts` and `source.ts`, are provided strictly as-is. LaunchDarkly Support cannot help run this.
```
Just a suggestion.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
 
-### Known Issues
+### Known issues
 
 - Bug with 429 rate limiting. The request that is rate limited will still fail
```
Comment (ember-stevens) [inline README.md]:
```suggestion
- Bug with 429 rate limiting. The rate-limited request will still fail,
```
I don't have much context for this section, so feel free to ignore any suggestions that don't make sense.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
-  but subsequent requests work
-- Importing LD API typescript types causes an import error so are commented out
-  in various spots
-- Types in generally are very loose which Deno is not happy about but it runs as
-  JavaScript overall compared to validating the TypeScript first
-- Due to the current API configuration, the environments count for a given project tops at 20 environments for a single project
-- Due to considerations around many API requests at once - monitor 400 errors for flag configurations that may not be up to date
-- To Avoid a race condition, a few waits have been placed in the script
+  but subsequent requests work.
```
Comment (ember-stevens) [inline README.md]:
```suggestion
  but subsequent requests will work.
```

File: README.md [hunk 4]
```diff
@@ -1,30 +1,32 @@
...
+- Importing LD API TypeScript types causes an import error, so are commented out
```
Comment (ember-stevens) [inline README.md]:
```suggestion
- Importing LD API TypeScript types causes an import error, so they are commented out
```

File: README.md [hunk 5]
```diff
@@ -1,30 +1,32 @@
...
+  in various spots.
+- Types in general are very loose, which Deno is not happy about. The scripts run as
+  JavaScript overall compared to validating the TypeScript first.
```
Comment (ember-stevens) [inline README.md]:
```suggestion
  JavaScript overall instead of validating the TypeScript first.
```

File: README.md [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
+- Due to the current API configuration, the environments count for a given project tops at 20 environments for a single project.
```
Comment (ember-stevens) [inline README.md]:
```suggestion
- Due to the current API configuration, you cannot have more than 20 environments in a single project.
```

File: README.md [hunk 7]
```diff
@@ -1,30 +1,32 @@
...
+- Due to considerations around many API requests at once, monitor 400 errors for flag configurations that may not be up to date.
```
Comment (ember-stevens) [inline README.md:21]:
```suggestion
- Due to considerations around many API requests at once, monitor 400 errors for flag configurations that may not be up to date.
```
I wonder what we mean by "considerations" here.

**Review by ember-stevens — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (mmrj) [reply README.md:21]: I think it's just another reference to the 429 (rate-limiting) errors.

Comment (cgreenld) [reply README.md:21]: Yeah, we had an issue around some bad request types - the script will bubble up 400 errors for specific calls, but continue to run.  Wanted to highlight that the output is designed to be helpful, if things don't look right

**Pull request merged**